src/OrchardCore.Modules/OrchardCore.Lists/OrchardCore.Lists.csproj

```diff
@@ -25,7 +25,8 @@
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.MetaWeblog.Abstractions\OrchardCore.MetaWeblog.Abstractions.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Module.Targets\OrchardCore.Module.Targets.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Users.Abstractions\OrchardCore.Users.Abstractions.csproj" />   
-    <ProjectReference Include="..\..\OrchardCore\OrchardCore.ResourceManagement\OrchardCore.ResourceManagement.csproj" />
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.ResourceManagement\OrchardCore.ResourceManagement.csproj" />   
+    <ProjectReference Include="..\OrchardCore.Title\OrchardCore.Title.csproj" />
```

Comment:
Ez a reference minek kell?

Reply:
Nem tudom mikor csináltam, resource már javítva, de lehet hogy title se fog kelleni, az a titlepart miatt lehetett

Reply:
Ez továbbra is ott van.

Reply:
A title kitörlése után nem volt gond, de a Resource után egy csomó hibát kaptam több fájlból is, amik az IResourceManager-t  és ResourceManager-t használják. Amikről nem tudom mennyire szükségesek, pl ListPart.RemotePublishing.cshtml vagy ListPartFeed.cshtml

Reply:
A title-t töröltem, gondolom csak az kell, mert a Resource benne volt az eredetiben is

Reply:
Igen, a Title volt a változás, szóval arra commenteltem.